### PR TITLE
Add model evaluation table into model zoo design

### LINKED ISSUE #1175: Model zoo also need to save evaluation info
**Is your feature request related to a problem? Please describe.**

Currently, the design of model zoo plans to use a table in a database to record model training info. It would be better to record model evaluation info, as users will be interested in how good the models in model zoo behave on open dataset. In my opinion, the model evaluation info includes model id, evaluation data, and evaluation metrics.

**Describe the solution you'd like**

As discussed offline with @wangkuiyi, we prefer to add an evaluation model zoo table in database.

**Describe alternatives (optional)**

None

**Additional Notes**

None

### PR DESCRIPTION
**Pull request by wangkuiyi**
Fix https://github.com/sql-machine-learning/sqlflow/issues/1175

### CODE REVIEW

**Comment by mhaoli** [comment]
LGTM!

File: doc/design/model_zoo.md [hunk 1]
```diff
@@ -99,7 +99,11 @@ The training submitter program `my_first_model.py`, running in the model definit
 
 ### Model Zoo Data Schema
 
-The model zoo table is in a database deployed as part of the SQLFlow service. This database might not be the one that holds data sources.  The only requirement of the model zoo table is to have a particular data schema that contains at least the following fields.
+The model zoo table is in a database deployed as part of the SQLFlow service. This database might not be the one that holds data sources.  The only requirement of the model zoo database is to have two tables: the **trained models table** and the **evaluation result table**.
+
+#### Trained Models
+
+Once a training job completes, the submitter program adds/updates a row of the the trained models table, which contains (at least) the following fields.
```
Comment (typhoonzero) [inline doc/design/model_zoo.md:106]: Note that if a saved model is large we may have multiple rows, each row stores part of the saved model.

**Review by typhoonzero — APPROVED** [review]

File: doc/design/model_zoo.md (hunk 1 above)
Comment (weiguoz) [reply doc/design/model_zoo.md:106]: How about saving the whole model to storage media like OSS?

Comment (typhoonzero) [reply doc/design/model_zoo.md:106]: Maybe for cases like training with PAI Tensorflow, we can store the URI of the saved model file on OSS in the "trained model table"?

**Pull request merged**